testing/docker/fedora30/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Building an SCons Release Build image under Fedora 30
+FROM fedora:30
```

Comment:
Is your idea that we will always synch these Docker files to the latest version of each OS? If yes, who will be responsible for keeping track of this? I could add another folder for fedora32 in a following PR if you need it right now, but I don't plan to keep pace with any release cycles. If anyone out there needs a new OS supported, the folder structure should be easy enough to use one of the existing setups as template (copy-paste)...ideally he/she would then contribute this to the project.

Reply:
I'd say (if we can) files here shouldn't be for EOL'd distro versions? is 30 eol'd?

Reply:
After 1 year only (what I found is that it was released in 2019, but this may be blatantly wrong)? Sorry, but my internal clock may be off and not used to modern release cycles yet. At work (ATC) I'm still working with SLES11 and ClearCase 7 (not UCM!) all day. So much for using no EOL versions. ;)

Reply:
However, if we find that Fedora 30 is EOL, I can understand your argument and wouldn't be opposed to amending the pull request. I'm just saying "can't we have both at the moment"? I simply like the idea of also having an OS as a "floor" version, where not all the packages, e.g. Python, are "new and shiny"...

Reply:
EOL was reached on 2020-05-26, bumped up to v32 as requested.